testdata/dnn/darknet/convolutional.cfg

```diff
@@ -0,0 +1,15 @@
+[net]
+batch=1
+channels=3
+height=4
+width=4
+
+#for SAME padding set pad=0, for manual padding set padding to desired value.
+[convolutional]
+filters=4
+size=3
+stride=2
+pad=1
+activation=leaky
```

Comment:
Can you also add a layer with `padding = 1` from https://github.com/opencv/opencv_extra/pull/697/ after it?
like
```
#Convolution layer
[convolutional]
filters=5
size=3
stride=1
padding=1
activation=leaky
```

Reply:
@dkurt Done!

Reply:
@dkurt Done!